ansible_collections/arista/avd/molecule/digital_twin/inventory/hosts.yml

```diff
@@ -26,3 +26,15 @@ all:
             # eAPI not enabled in default VRF. Automatically enabling it.
             digital-twin-enforce-eapi-4:
               type: l2leaf
+            # OOB MGMT is already Ma1 (by default). ACT Digital Twin version of the config respects it.
```

Comment:
Please move these commments to the files so when we read the vars we understand what you are testing. Thanks